players/installing-fabric.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Installing Fabric
+description: A step by step guide on how to install Fabric.
+---
+
+# Installing Fabric
+
+This guide will walk you through installing Fabric for the Minecraft Launcher.
+
+For third party launchers, you should consult their documentation.
+
+## 1. Download the Fabric Installer
+
+You can download the Fabric Installer from the [Fabric Website](https://fabricmc.net/use/).
+
+For windows, it's recommended you download the `.exe` version (`Download For Windows`), as it doesn't require java to be installed on your system (it will use the java installation that comes with the Minecraft Launcher).
+
+For macOS and Linux, you should download the `.jar` version, this may mean you need to install java firstly though.
+
+## 2. Run the Fabric Installer
+
+::: warning
+Before running the installer, you must close the Minecraft Launcher if it is running, along with any other Minecraft instances that are currently running.
```

Comment:
I think this is fine to keep, however the installer will show a warning if you try to install when the launcher is open